server/lib/renderers/xlsxReceipt.js

```diff
@@ -0,0 +1,207 @@
+  ws.cell(2, 1, 2, 3, true).string(t('FORM.LABELS.ADDRESS').concat(': ', data.enterprise.location)).style(style2);
+  ws.cell(3, 1, 3, 3, true).string(t('FORM.LABELS.PHONE').concat(': ', data.enterprise.phone)).style(style2);
+  ws.cell(4, 1, 4, 3, true).string(t('FORM.LABELS.EMAIL').concat(': ', data.enterprise.email)).style(style2);
+
+  // cell(1,6)=> F1
+  ws.cell(1, 6).string(t('FORM.LABELS.INVOICE')).style(style1);
+  ws.cell(2, 6).string(data.reference).style(style1);
+  ws.cell(3, 6).string(data.dateFormat).style(style2);
+
+  // cell(7, 1) => A7
+  ws.cell(7, 1, 7, 3, true).string(t('FORM.LABELS.CLIENT').concat(': ', data.recipient.reference));
+  ws.cell(8, 1, 8, 3, true).string(t('FORM.LABELS.NAME').concat(': ', data.recipient.display_name));
+  ws.cell(9, 1, 9, 3, true).string(t('FORM.LABELS.GROUP').concat(': ', data.recipient.debtor_group_name));
+  ws.cell(10, 1, 10, 3, true).string(t('FORM.LABELS.HOSPITAL_FILE_NR').concat(': ', data.recipient.hospital_no));
+
+
+  ws.cell(7, 5, 7, 7, true).string(t('FORM.LABELS.INVOICE').concat(': ', data.reference));
+  ws.cell(8, 5, 8, 7, true).string(t('TABLE.COLUMNS.SERVICE').concat(': ', data.serviceName));
+  ws.cell(9, 5, 9, 7, true).string(t('FORM.LABELS.DATE').concat(': ', data.dateFormat));
+  ws.cell(10, 5, 10, 7, true).string(t('FORM.LABELS.CREATED_BY').concat(': ', data.display_name));
+
+  ws.cell(7, 1, 10, 7).style(style2);
+
+
+  ws.cell(7, 1, 7, 7).style(styleBorderTop);
+
+  ws.cell(10, 1, 10, 7).style(styleBorderBottom);
+  ws.cell(7, 1, 10, 1).style(styleBorderLeft);
+  ws.cell(7, 7, 10, 7).style(styleBorderRight);
+
+  // Description
+
+  ws.cell(12, 1).string(t('FORM.LABELS.DESCRIPTION')).style(styleUndeLine);
+  ws.cell(13, 1, 13, 7, true).string(data.description).style(style2);
+  ws.cell(13, 1, 13, 7, true).style(styleMultiLine);
+  ws.row(13).setHeight(40); // description might be pretty long
+
+  // detail
+  ws.cell(15, 1).string(t('FORM.LABELS.INVOICES_DETAILS')).style(styleUndeLine);
+
+  // titles
+  let line = 16;
+  ws.cell(line, 1).string(t('TABLE.COLUMNS.CODE'));
+  ws.cell(line, 2, line, 4, true).string(t('TABLE.COLUMNS.DESCRIPTION'));
+  ws.cell(line, 5).string(t('TABLE.COLUMNS.UNIT_PRICE'));
+  ws.cell(line, 6).string(t('TABLE.COLUMNS.QUANTITY'));
+  ws.cell(line, 7).string(t('TABLE.COLUMNS.TOTAL'));
+  ws.cell(line, 1, line, 7).style(style2);
+  ws.cell(line, 1, line, 7).style(styleAllBorders);
+
+  line = 17;
+  // invoice items
+  data.items.forEach(row => {
+    ws.cell(line, 1).string(row.code);
+    ws.cell(line, 2, line, 4, true).string(row.text);
+    ws.cell(line, 5).number(row.transaction_price);
+    ws.cell(line, 6).number(row.quantity);
+
+    const total = (row.transaction_price * row.quantity);
+    ws.cell(line, 7).string(''.concat(total, '', currencySymbol));
+
+    ws.cell(line, 1, line, 7).style(style2);
+    ws.cell(line, 1, line, 7).style(styleAllBorders);
+    line++;
```

Comment:
Instead of doing `line++`, you could also use an offset:
```js
const offset = 17;
data.items.forEach((row, idx) => {
  const line = idx + offset;
  // other code...
});
```

Reply:
```line``` is used in many places, not only here